dataplane/worker_gc.h

```diff
@@ -39,6 +39,7 @@ class worker_gc_t
 	void handle_free_mbuf();
 
 	bool is_timeout(const uint16_t timestamp, const uint16_t timeout);
+	uint16_t get_acl_tcp_timeout(const uint8_t flags, const uint32_t ack_timeout, const uint32_t syn_timeout, const uint32_t fin_timeout, const uint32_t default_timeout);
```

Comment:
Having function with 5 arguments seems alien. I'd rather obtained concrete timeouts inside the function using `globalbase_atomic` directly.